From 9b948db8377c0ac146eae2d865c987e73d9109ad Mon Sep 17 00:00:00 2001
From: Mamoru TASAKA <mtasaka@fedoraproject.org>
Date: Fri, 11 Dec 2020 10:40:09 +0900
Subject: [PATCH] test-screens.c: add skel XA_SCREENSAVER_VERSION definition

To avoid the following link error:
----------------------------------------------------
/usr/bin/ld: test-screens.o: in function `update_screen_layout':
/builddir/build/BUILD/xscreensaver-5.45/x86_64-pc-linux-gnu/driver/../../driver/screens.c:1104: undefined reference to `XA_SCREENSAVER_VERSION'
collect2: error: ld returned 1 exit status
----------------------------------------------------

On xscreensaver-5.45, in screens.c/update_screen_layout some logic is added using
XA_SCREENSAVER_VERSION .
---
 driver/test-screens.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/driver/test-screens.c b/driver/test-screens.c
index 2fb3e35..c5dfa02 100644
--- a/driver/test-screens.c
+++ b/driver/test-screens.c
@@ -39,6 +39,7 @@
 
 char *progname = 0;
 char *progclass = "XScreenSaver";
+Atom XA_SCREENSAVER_VERSION;
 
 const char *blurb(void) { return progname; }
 
-- 
2.29.2

